dhis_android_analytics/src/main/java/dhis2/org/analytics/charts/mappers/GraphToBarData.kt

```diff
@@ -38,6 +39,36 @@ class GraphToBarData {
                     color = serieColor
                 }
             }
-        ).withGlobalStyle()
+        ).withGlobalStyle(barWidthByPeriod(graph))
+    }
+
+    private fun barWidthByPeriod(graph: Graph): Float? {
+        return when (graph.eventPeriodType) {
+            PeriodType.Daily ->
+                1f / 30f
```

Comment:
Why don't we create descriptive constants instead of magic numbers?. Like:
`private const val BAR_WIDTH_DAILY = 1f / 30f`